hack/verify-api-groups.sh

```diff
@@ -98,7 +98,14 @@ packages_without_install=(
 )
 known_version_files=(
 	"pkg/master/import_known_versions.go"
-	"pkg/client/clientset_generated/internal_clientset/import_known_versions.go"
+# this file doesn't seem to exist
+# we were not actually checking it previously, due to a bug in the script
+#	"pkg/client/clientset_generated/internal_clientset/import_known_versions.go"
```

Comment:
@deads2k it looks like you [originally wrote this](https://github.com/kubernetes/kubernetes/commit/4850314a5cee4064e7d1b7f23ecc14232c7c3eee), do you know what the correct change would be here?

because of the bug below, we were only ever checking `pkg/master/import_known_versions.go`
(https://github.com/kubernetes/kubernetes/pull/73061/files#r248936285)

Reply:
discussed in #sig-api-machinery slack [here](https://kubernetes.slack.com/archives/C0EG7JC6T/p1547836209407000), it looks like we can just remove this check